tests/standardise/test_standardise.py

```diff
@@ -272,24 +272,38 @@ def test_local_obs_metadata_mismatch_fail():
 
 
 def test_standardise_column():
+    """
+    Tests standardise column function and associated metadata keys
+    for satellite column data.
+    """
     filepath = get_column_datapath(filename="gosat-fts_gosat_20170318_ch4-column.nc")
 
     satellite = "GOSAT"
-    domain = "BRAZIL"
+    domain = "SOUTH-AMERICA"
+    selection = "LAND"
     species = "methane"
+    region = "BRAZIL"
 
     results = standardise_column(
         filepath=filepath,
         source_format="OPENGHG",
         satellite=satellite,
         domain=domain,
         species=species,
+        region=region,
+        selection=selection,
         force=True,
         store="user",
     )
 
     assert "ch4" == results[0].get("species")
 
+    data = get_obs_column(species='ch4', max_level=3)
+
+    assert data.metadata["region"] == "brazil"
+    assert data.metadata["selection"] == "land"
+    assert data.metadata["domain"] == "south-america"
```

Comment:
```suggestion
    assert data.metadata["domain"] == "southamerica"
```

Reply:
Actually - given the above suggestion for how we use these keywords - we'll likely want to remove `domain` from this and only include `obs_region`.